src/scripts/main.js

```diff
@@ -1,3 +1,8 @@
 'use strict';
 
-// write your code here
+const tr = document.querySelectorAll('tr');
```

Comment:
```suggestion
const rows = document.querySelectorAll('tr');
```
it's better to use more meaningful variable names